packages/router/router.ts

```diff
@@ -790,12 +813,13 @@ export function createRouter(init: RouterInit): Router {
     }
 
     // Call action if we received an action submission
-    let pendingActionData: RouteData | null = null;
-    let pendingActionError: RouteData | null = null;
+    let request = createRequest(location, opts?.submission);
```

Comment:
This is the big change to current router internals in here.  In order to better align the server/browser use-cases we life `Request` creation way up from where it used to be.  That way we're better able to accept an external `Request` ion the SSR case